src/view/components/DataTable/index.tsx

```diff
@@ -34,8 +35,8 @@ export default class DataTable extends React.Component<PropsType, OwnStateType>
   private formatValues(values: any[]) {
     return values.map((value) => {
       return (
-        <Col key={value}>
-          <Text>
+        <Col key={value} style={{ justifyContent: 'center' }}>

```

Comment:
Should we move this into a stylesheet?

Reply:
Since there were a bunch of styles that were already defined inline, I created a maintenance ticket to move all inline styles to a stylesheet and to replace all hardcoded hex strings (see https://github.com/agriculture-capstone/agritrader/issues/120) 

I just tried to get the styles working as fast as I could so I could help out with the backend soon :)